src/main/kotlin/com/openlattice/organizations/tasks/OrganizationsInitializationTask.kt

```diff
@@ -50,8 +50,9 @@ class OrganizationsInitializationTask : HazelcastInitializationTask<Organization
         logger.info("Running bootstrap process for organizations.")
         val sw = Stopwatch.createStarted()
         val organizationService = dependencies.organizationService
+        val partitionManager = dependencies.partitionManager
         val globalOrg = organizationService.maybeGetOrganization(GLOBAL_ORG_PRINCIPAL)
-        val defaultPartitions = organizationService.allocateDefaultPartitions(organizationService.numberOfPartitions)
+        val defaultPartitions = partitionManager.getAllPartitions()
```

Comment:
Are you making it so that global organization entity sets are by default spread across all partitions?

Reply:
Yes but this code should be equivalent to the previous call, just a bit clearer IMO